l10n_it_website_portal_fatturapa/views/l10n_it_website_portal_fatturapa_templates.xml

```diff
@@ -0,0 +1,76 @@
+            <script type="text/javascript" src="/l10n_it_website_portal_fatturapa/static/src/js/l10n_it_website_portal_fatturapa.js"/>
+        </xpath>
+    </template>
+
+    <template id="fatturapa_details" name="Add fatturapa fields" inherit_id="l10n_it_website_portal_fiscalcode.fiscalcode_details">
+        <xpath expr="//input[@name='name']/.." position="attributes">
+            <attribute name="t-attf-class" separator=" " add="#{'o_hidden' if not partner.is_company else ''}"/>
+        </xpath>
+        <xpath expr="//input[@name='name']/.." position="after">
+            <t t-if="not partner.is_company">
+                <div class="clearfix"/>
+                <div t-attf-class="form-group #{error.get('firstname') and 'has-error' or ''} col-lg-6">
+                    <label class="control-label" for="firstname">
+                        Your First Name
+                    </label>
+                    <input type="text" name="firstname" class="form-control"
+                           t-att-value="firstname or partner.firstname"/>
+                </div>
+                <div t-attf-class="form-group #{error.get('lastname') and 'has-error' or ''} col-lg-6">
+                    <label class="control-label" for="lastname">
+                        Your Last Name
+                    </label>
+                    <input type="text" name="lastname" class="form-control"
+                           t-att-value="lastname or partner.lastname"/>
+                </div>
+                <div class="clearfix"/>
+            </t>
+        </xpath>
+        <xpath expr="//input[@name='fiscalcode']/.." position="after">
+            <div class="clearfix"/>
+            <div t-attf-class="form-group #{error.get('codice_destinatario') and 'has-error' or ''} col-lg-6">
+                <label class="control-label label-optional" for="codice_destinatario">
+                    Your Addressee code
+                </label>
+                <input type="text" name="codice_destinatario" class="form-control"
+                       t-att-value="codice_destinatario or partner.codice_destinatario"/>
+            </div>
+            <div t-attf-class="form-group #{error.get('pec_destinatario') and 'has-error' or ''} col-lg-6">
+                <label class="control-label label-optional" for="pec_destinatario">
+                    Your PEC address for Electronic invoicing
+                </label>
+                <input type="text" name="pec_destinatario" class="form-control"
+                       t-att-value="pec_destinatario or partner.pec_destinatario"/>
+            </div>
+            <div class="clearfix"/>
+            <div t-attf-class="form-group #{error.get('is_pa') and 'has-error' or ''} col-lg-6">
+                <label class="control-label label-optional" for="is_pa">
+                    Are you acting on behalf of a Public Administration?
+                </label>
+                <input type="checkbox" name="is_pa" class="form-control"
```

Comment:
Se è una PA, è anche una company (vedi backend)

Reply:
Tolgo il campo is_pa perché va in conflitto con il fatto che gli utenti di portale sono sempre pensati come individui, lo aggiungo alla roadmap